src/CBLReplicatorConfig.hh

```diff
@@ -196,6 +208,14 @@ namespace cbl_internal {
 
         ~ReplicatorConfiguration() {
             release(database);
+            
+            if (collections) {
+                for (int i = 0; i < collectionCount; i++) {
+                    release(collections[i].collection);
+                }
+                delete [] collections;
```

Comment:
Using a vector avoids having to do the array delete.

Reply:
Actually I can use vector here too.